File: branches/Medusa/Source/Fantasy.Jobs/Resources/TemplateRunTimeScheduleSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;
using Fantasy.Configuration;

namespace Fantasy.Jobs.Resources
{
    public class TemplateRuntimeScheduleSettings : RuntimeScheduleSetting
    {
        private List<TemplateRuntimeScheduleSetting> _templates = new List<TemplateRuntimeScheduleSetting>();

        [XmlArray("templates"),
        XmlArrayItem(ElementName="template", Type = typeof(TemplateRuntimeScheduleSetting))]
        public List<TemplateRuntimeScheduleSetting>  Templates
        {
            get { return _templates; }
        }



      
    }


    public class TemplateRuntimeScheduleSetting : RuntimeScheduleSetting
    {

        [XmlAttribute("name")]
        public string Name { get; set; }



       
    }
}

File: AutoConverted/Fantasy.Jobs/Resources/TemplateRuntimeScheduleSettings.java
package Fantasy.Jobs.Resources;

import Fantasy.Configuration.*;

public class TemplateRuntimeScheduleSettings extends RuntimeScheduleSetting
{
	private java.util.ArrayList<TemplateRuntimeScheduleSetting> _templates = new java.util.ArrayList<TemplateRuntimeScheduleSetting>();

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XmlArray("templates"), XmlArrayItem(ElementName="template", java.lang.Class = typeof(TemplateRuntimeScheduleSetting))]
	public final java.util.ArrayList<TemplateRuntimeScheduleSetting> getTemplates()
	{
		return _templates;
	}




}